pymc_marketing/mlflow.py

```diff
@@ -446,6 +803,9 @@ def autolog(
         Additional keyword arguments to pass to `az.summary`.
     log_mmm : bool, optional
         Whether to log PyMC-Marketing MMM models. Default is True.
+    log_loocv : bool, optional
+        Whether to log LOOCV metrics for model. Default is False.
+        Requires the user to have already called `pm.compute_log_likelihood()` on the model.
```

Comment:
Think this logic is a bit tangled. We need to call the compute_log_likelihood before calling `pm.sample` / `MMM.fit` which cannot be done

It might be best to patch the `pm.compute_log_likelihood` directly then it can be used in arbitrary PyMC models as well 😄

Reply:
I've pushed a change for this - but I'm not fully happy with it. I didn't think about `mmm.fit()` calls `pm.sample()` internally, so you run the risk of creating a log_likelihood_group twice when you do mmm autologging.

My solution just checks that there isn't already a log-likelihood group present, but this seems a bit of a hacky fix

Reply:
Feel free to lemme know if you can think of a better way on this @wd60622! Otherwise, that should hopefully be the comments addressed now ☺️

Reply:
Looks fine to me. Thanks for the changes